api/src/main/scala/play/twirl/api/Content.scala

```diff
@@ -57,4 +57,11 @@ abstract class BufferedContent[A <: BufferedContent[A]](protected val elements:
 
   def body = builtBody
 
+  override def equals(obj: Any): Boolean = obj match {
+    case other: BufferedContent[_] => body == other.body
```

Comment:
Probably should make this also check that they are the same class, ie `other.getClass == this.getClass`, because `Xml("foo")` probably shouldn't equal `Html("foo")`.

A few tests would also be good.